src/vhost.c

```diff
@@ -472,11 +386,106 @@ static void destroy_vm(int dev)
 			break;
 		}
 	}
+	pthread_mutex_unlock(&mutex);
+}
+static void vhost_create_socket(struct pg_vhost_state *state, uint64_t flags,
```

Comment:
why did you change the position of thoses function ?
Problem is that if you change things in them, it doesn't appear in the changelog